editor/changeset_wrapper.hpp

```diff
@@ -29,7 +31,8 @@ class ChangesetWrapper
 
   /// Throws many exceptions from above list, plus including XMLNode's parsing ones.
   /// OsmObjectWasDeletedException means that node was deleted from OSM server by someone else.
-  editor::XMLFeature GetMatchingFeatureFromOSM(editor::XMLFeature const & ourPatch, FeatureType const & feature);
+  editor::XMLFeature GetMatchingNodeFeatureFromOSM(m2::PointD const & center);
```

Comment:
Ping on this comment - I do not see forward declaration or inclusion of geometry - point2d.